lumicks/pylake/kymo.py

```diff
@@ -380,6 +408,35 @@ def pixelcount_factory(_):
         result._pixelcount_factory = pixelcount_factory
         return result
 
+    def calibrate_to_kbp(self, length_kbp):
+        """Calibrate from microns to other units.
+
+        Parameters
+        ----------
+        length : int
+            length of the kymo in kilobase pairs
+        """
+        if self._calibration.unit == "kbp":
+            raise RuntimeError("kymo is already calibrated in base pairs.")
+
+        calibration = PositionCalibration("kbp", length_kbp / self.size_um[0], "kbp")
+        result = Kymo(
+            self.name,
+            self.file,
+            self.start,
+            self.stop,
+            self._json,
+            self._position_offset,
+            calibration,
+        )
+        result._image_factory = self._image_factory
+        result._timestamp_factory = self._timestamp_factory
+        result._line_time_factory = self._line_time_factory
+        result._pixelsize_factory = self._pixelsize_factory
+        result._pixelcount_factory = self._pixelcount_factory
```

Comment:
Not suggesting to piggyback it on this PR, but feels like we're a bit overdue for a copy method.

Reply:
we have a ticket open for that, I pushed it up a bit in the backlog